src/thing_creature.c

```diff
@@ -5435,6 +5450,7 @@ TngUpdateRet update_creature(struct Thing *thing)
     cctrl->flgfield_1 &= ~CCFlg_Unknown80;
     cctrl->spell_flags &= ~CSAfF_PoisonCloud;
     process_thing_spell_effects(thing);
+    process_timebomb(thing);
```

Comment:
It's a separate thing compared to process_thing_spell_effects now. I think this is the reason why reapply_spell_effect_to_thing does not work on TIMEBOMB. I think it should work and reset the timer.

Reply:
The creature does not even freeze to let you try to reapply the spell. How can this be dealt with?

Reply:
If I revert 4aebd9f8a1a93f5d35c90a4733bef10d53792c5a, Timebombed creatures stop to let you do stuff, but I can also pick them up.

Reply:
It seems that only creatures whom you can pick up stop to let you do things when the cursor is near, but I can't find the code that causes this...